src/crab/ebpf_transformer.cpp

```
@@ -588,39 +589,28 @@ void EbpfTransformer::do_load(const Mem& b, const Reg& target_reg) {
-        }
-        case T_PACKET: {
-            LinearExpression addr = mem_reg.packet_offset + offset;
-            do_load_packet_or_shared(state, target_reg, width, b.is_signed);
-            break;
-        }
-        case T_SHARED: {
-            LinearExpression addr = mem_reg.shared_offset + offset;
-            do_load_packet_or_shared(state, target_reg, width, b.is_signed);
-            break;
-        }
+        case T_CTX: do_load_ctx(state, context, target_reg, mem_reg.ctx_offset + offset, width); break;
```

Clang (14.0.6)

[warning] switch has 2 consecutive identical branches

(bugprone-branch-clone)